website/docs/reverse-proxy.md

```diff
@@ -57,6 +57,35 @@ Apache virtual server configuration.
     </IfModule>
 ````
 
+## Invalid checksum
+
+Sometimes the gzip compression can mess with the request when running `npm install` and result in error messages like this:
```

Comment:
We might link the issue you filled in case someone have similar corner cases.

Reply:
Good idea.